test/com/facebook/buck/testutil/MoreAssertsTest.java

```diff
@@ -25,14 +25,9 @@
 
 public class MoreAssertsTest {
 
-  @SuppressWarnings("PMD.EmptyCatchBlock")
   private void expectFail(Iterable<?> a, Iterable<?> b) {
-    try {
-      MoreAsserts.assertIterablesEquals(a, b);
-      fail("Expected failure from: a=" + a + "; b=" + b);
-    } catch (AssertionError e) {
-      // OK
-    }
+    MoreAsserts.assertIterablesEquals(a, b);
```

Comment:
I've sent out a change fixing this in a way which actually fixes the test. Please revert this bit of the change.